benchmarks/README.md

```diff
@@ -44,8 +46,86 @@ Once the EMR cluster is ready, note the following:
 
-### Prepare S3 bucket
-Create a new S3 bucket (or use an existing one) which is in the same region your EMR cluster.
+#### Prepare S3 bucket
+Create a new S3 bucket (or use an existing one) which is in the same region as your EMR cluster.
+
+_________________
+
+### Google Cloud Platform
+
+#### Prerequisites
+- A GCP account with necessary permissions to do the following:
+  - Manage Dataproc clusters for running the benchmark
+  - Manage Dataproc Metastore instances
+  - Read and write to a GCS bucket from the Dataproc cluster
+- A GCS bucket which will be used to generate the TPC-DS data.
+- A machine which has access to the GCP setup and where this repository has been downloaded or cloned.
+- SSH keys for a user which will be used to access the master node. The user's SSH key can be either [a project-wide key](https://cloud.google.com/compute/docs/connect/add-ssh-keys#add_ssh_keys_to_project_metadata) 
+  or assigned to the [master node](https://cloud.google.com/compute/docs/connect/add-ssh-keys#after-vm-creation) only.
+- Ideally, all GCP components used in benchmark should be in the same location (Storage bucket, Dataproc Metastore service and Dataproc cluster).
+
+#### Prepare GCS bucket
+Create a new GCS bucket (or use an existing one) which is in the same region as your Dataproc cluster.
+
+#### Create Dataproc Metastore
+You can create [Dataproc metastore](https://cloud.google.com/dataproc-metastore/docs/create-service)
+either via Web Console or gcloud command.
+
+Sample create command:
+```bash
+gcloud metastore services create dataproc-metastore-for-benchmarks \
+    --location=<region> \
+    --tier=enterprise
+```
+
+#### Create Dataproc cluster
+Here are the specifications of the Dataproc cluster required for running benchmarks.
+- Image version >= 2.0 having Apache Spark 3.1
+- Master - n2-highmem-8 (8 vCPU, 64 GB memory)
+- Workers - 16 x n2-highmem-8 (or just 2 workers if you are just testing by running the 1GB benchmark).
+- The cluster connects to the Dataproc Metastore.
+- Same region and subnet as those of the Dataproc Metastore and GCS bucket.
+- No autoscaling.
+
+Sample create command:
+```bash
+gcloud dataproc clusters create delta-performance-benchmarks-cluster \
+    --project <project-name> \
+    --enable-component-gateway \
+    --region <region> \
+    --zone <zone> \
+    --subnet default \
+    --master-machine-type n2-highmem-8 \
+    --master-boot-disk-type pd-ssd \
+    --master-boot-disk-size 100 \
+    --num-master-local-ssds 4 \
+    --master-local-ssd-interface NVME \
+    --num-workers 16 \
+    --worker-machine-type n2-highmem-8 \
+    --worker-boot-disk-type pd-ssd \
+    --worker-boot-disk-size 100 \
+    --num-worker-local-ssds 4 \
+    --worker-local-ssd-interface NVME \
+    --dataproc-metastore projects/<project-name>/locations/<region>/services/dataproc-metastore-for-benchmarks \
+    --enable-component-gateway \
+    --image-version 2.0-debian10
+```
+
+#### Input data
```

Comment:
Now I thinking about having a permanent copy of TPC-DS data in Databricks' Google Storage bucket, marked as "Requester Pays", until Google Dataproc supports Hadoop >= 3.3.4, instead of asking users to copy the data to their buckets first. WDYT @dennyglee ?

Reply:
This is a good idea but let's create a new issue to put the data into an open GCS bucket